api/types/accesslist/convert/v1/accesslist.go

```diff
@@ -143,6 +143,13 @@ func FromProto(msg *accesslistv1.AccessList, opts ...AccessListOption) (*accessl
 		MemberOf:        memberOf,
 	}
 
+	if msg.Status != nil && msg.Status.SelfAccess != nil {
```

Comment:
```suggestion
	if msg.GetStatus().SelfAccess != nil {
```

bonus point for 
```
accessList.Status = convertToStatus(msg.Status)
```


Additionally   we should keep this in call closer to [ line 96](https://github.com/gravitational/teleport/blob/7f85724d4d35cf115d488b0a4eb3b1b95dcfafb0/api/types/accesslist/convert/v1/accesslist.go#L95) where other Status fields are set.